### PR TITLE
Fix UI dependencies versions changed in license eye

### PR DESCRIPTION
**Pull request by kezhenxu94**
- [ ] If this pull request closes/resolves/fixes an existing issue, replace the issue number. Closes #<issue number>.
- [ ] Update the [`CHANGES` log](https://github.com/apache/skywalking/blob/master/docs/en/changes/changes.md).

### CODE REVIEW

**Review by wu-sheng — APPROVED** [review]

**Comment by wu-sheng** [comment]
`CI / Dependency licenses` skipped, have you checked this locally?

**Comment by kezhenxu94** [comment]
> `CI / Dependency licenses` skipped, have you checked this locally?

this is expected, I verified this locally.

**Comment by kezhenxu94** [comment]
Just also enabled the dependencies check when UI changed

**Review by wu-sheng — APPROVED** [review]

**Comment by kezhenxu94** [comment]
> Just also enabled the dependencies check when UI changed

Looks like the action cannot detect specific files changes in sub module, just change to if ui sub module changed trigger the workflow.

**Comment by wu-sheng** [comment]
Make sense. Submodule is one file from git perspective.

File: .github/workflows/skywalking.yaml [hunk 1]
```diff
@@ -136,12 +136,14 @@ jobs:
               }"
             pom:
               - "**/pom.xml"
+            ui:
+              - "skywalking-ui/**"
```
Comment (wu-sheng) [inline .github/workflows/skywalking.yaml:140]: Folder seems not working too.

Comment (wu-sheng) [reply .github/workflows/skywalking.yaml:140]: Maybe we should try 
```suggestion
              - "skywalking-ui"
```

**Comment by wu-sheng** [comment]
Look like dependencies check still not running

**Comment by wu-sheng** [comment]
Still `CI / Dependency licenses (pull_request) Skipped` :(

**Review by wu-sheng — APPROVED** [review]

**Pull request merged**